### PR TITLE
Add release note for 2.5.0

### PR DESCRIPTION
**Pull request by mingshl**
### Description
Add release notes for 2.5.0

### CODE REVIEW

File: release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.5.0
+### 🛡 Security
+* Update yarn.lock ([#107](https://github.com/opensearch-project/dashboards-search-relevance/pull/107))([#113](https://github.com/opensearch-project/dashboards-search-relevance/pull/113))
```
[the remark this answers is not in the document]
Comment (mingshl) [reply release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md:3]: 107 and 113 are for the 3 cve security fix.

[the remark this answers is not in the document]
Comment (mingshl) [reply release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md:3]: notified in other repo, if multiple PR for the same issues, they are likely notified in the same bullet point, sample [here](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/ba1862c9cf1036e82f3a285dcadb72f26c6dd856/release-notes/opensearch-dashboards.release-notes-2.4.0.md?plain=1#LL35C65-L35C65)

[the remark this answers is not in the document]
Comment (noCharger) [reply release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md:3]: If you really want to do that, then please also refine the content to be more meaningful

[the remark this answers is not in the document]
Comment (macohen) [reply release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md:3]: I like keeping them on the same line. I also agree with making the descriptions more meaningful by saying why the pull request exists in the title. Moving forward, let's make PR titles part of the review process so this doesn't HAVE to be handled in the release notes.

[the remark this answers is not in the document]
Comment (mingshl) [reply release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md]: @noCharger @macohen rename the pr title #113 with `Bump version for ansi-regex, gs, glob-parent and update yarn.lock`. I hope this title make more sense for the 3 packages version update and yarn lock update all together.

[the remark this answers is not in the document]
Comment (msfroh) [reply release-notes/opensearch-dashboards-search-relevance.release-notes-2.5.0.0.md]: We don't need commit subject lines to explain *why* a commit is made. See https://cbea.ms/git-commit/ for guidance.

However, we do need the release notes to summarize things in a way that's relevant to readers.

**Review by noCharger — APPROVED** [review]
The new message is a good summary of both PRs. Approved

**Pull request merged**